modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc
+
+[id="nw-ovn-ipsec-disable_{context}"]
+= Disabling IPsec encryption
+
+As a cluster administrator, you can disable IPsec encryption.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To disable IPsec encryption, enter the following command:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io/cluster --type=merge -p \
+  '{
+    "spec": { "defaultNetwork": { "ovnKubernetesConfig": { "ipsecConfig": null }}}
+  }'
+----
```

Comment:
Hi @jboxman-rh, i don't have much comments  but maybe we can remove the IPsec config instead of change it to null: 
$ oc patch networks.operator.openshift.io/cluster --type=json -p='[{"op":"remove", "path":"/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig"}]'.

that's it lgtm.
Thanks

Reply:
Hi, @jboxman-rh, it looks like you're leaving this as is?